compiler-plugin/src/main/kotlin/arrow/meta/phases/analysis/ElementScope.kt

```diff
@@ -271,12 +271,6 @@ interface ElementScope {
   
   fun String.blockCodeFragment(context: PsiElement?): Scope<KtBlockCodeFragment>
   
-  fun `if`(
```

Comment:
I think this method makes more sense if instead of taking `KtExpression` values it takes our scope values instead